calc/src/test/calc.test.ts

```diff
@@ -208,6 +208,25 @@ describe('calc', () => {
       });
     });
 
+    describe('IVs are shown if applicable', () => {
```

Comment:
I changed the test [5a666c6](https://github.com/smogon/damage-calc/pull/604/commits/5a666c6719f1aed134410322815e4e301123e7bf) to be more direct. 

Funny thing is that if I change the evs to be all zero, I get different messages in different gens (gen 3 says 'possible 7HKO', all other gens say 'guarenteed 7hko'), so I kept the raw desc access but just directly checked each field instead of being fancy.
![image](https://github.com/smogon/damage-calc/assets/7078074/8b1298b8-5e1c-46b7-ade0-e8b0b77a42dc)
This is taken with the live calculator, so it doesn't show the evs, but it's sufficient to make the tests fail if I just use descriptions.

Reply:
Implemented in [3adf7b6](https://github.com/smogon/damage-calc/pull/604/commits/3adf7b6a662531e56a445a4f34d321c6a32d5a9c)